ai_worker/main.py

```diff
@@ -103,11 +106,36 @@ class WorkerMain:
     def __init__(self, conf: Config):
         self.__connect_info: Optional[ConnectMessage] = None
         self.conf = conf
+        self._gen_or_load_priv()
```

Comment:
worker_id should probably vary based on the --main_gpu selected.   include a pubkey, and optional "subkey"as well.  worker_id is the combination of both (if the subkey not pressen we assume subkey is blank).   if the user specified a main gpu and/or a set of tensor splits, we hash that and call it a subkey.   this allows multiple stats linked to gpu config.   and it allows the user to override the subkey to a "config name" or whatever.

Reply:
this is done